hosting/src/main.css

```diff
@@ -1,16 +1,43 @@
 @tailwind base;
 @tailwind components;
+@tailwind utilities;
 
-.tooltip {
-  @apply invisible absolute;
-}
+@layer utilities {
+  .tooltip {
+    @apply invisible absolute;
+  }
```

Comment:
All these @s are new to me too. I'm showing my css-age.

Reply:
I think this is tailwind magic